### PR TITLE
MES collector horn fix

### PR DESCRIPTION
**Pull request by ransty**
Fix as described by https://github.com/SkylerPIlot/Ba-Minigame/issues/1#issuecomment-1134183080

image

### CODE REVIEW

**Review by SkylerPIlot — APPROVED** [review]
LGTM

**Comment by SkylerPIlot** [comment]
Will wait for review from equips before merging though

File: gradlew
Comment (equirs) [reply gradlew:75]: Could be something to do with your OS, not sure

**Pull request merged**
